yt/frontends/arepo/data_structures.py

```diff
@@ -68,18 +68,20 @@ def _is_valid(cls, filename, *args, **kwargs):
     def _get_uvals(self):
         handle = h5py.File(self.parameter_filename, mode="r")
         uvals = {}
-        missing = [False] * 3
+        missing = [True] * 3
         for i, unit in enumerate(
             ["UnitLength_in_cm", "UnitMass_in_g", "UnitVelocity_in_cm_per_s"]
         ):
-            if unit in handle["/Header"].attrs:
-                uvals[unit] = handle["/Header"].attrs[unit]
-                if unit == "UnitLength_in_cm":
-                    # We assume this is comoving, because in the absence of comoving
-                    # integration the redshift will be zero.
-                    uvals["cmcm"] = 1.0 / uvals[unit]
-            else:
-                missing[i] = True
+            for grp in ["/Header", "/Parameters", "/Units"]:
```

Comment:
This is nice!

Reply:
I am completely stumped as to the Jenkins test failures. I cannot reproduce the difference on this PR vs. `main` on my own machine--even after switching to Python 3.8 and letting pip install all of the dependencies. Can someone help check the logic here to make sure I didn't do something silly?

Reply:
FWIW, both answers I get (this PR vs `main` on my machine) agree with the "new" ones

Reply:
^^ @Xarthisius any special Jenkins insight would be helpful